### PR TITLE
Add UCI Adult example

### PR DESCRIPTION
**Pull request by liufuyang**
The code gives acc score 0.830.
I tried it with the old version, it gives 0.829 (And by changing the default pseudo count to 0.1 I got 0.8302929795467109 again)

Which might be a little weird, when comparing with my Rust implementation, I got 0.833 acc with an implementation which should be very close to the old version blayze, so I was expecting the score to be around 0.833 🤔 ...

### CODE REVIEW

**Comment by liufuyang** [comment]
image


```
@Test
    fun can_fit_uci_adult_dataset() {
        val train = uciAdult("adult.train.txt")
        val test = uciAdult("adult.test.txt")

        val index = arrayListOf(0, 5, 10, 15, 20, 25, 30, 40, 50, 60, 80, 100, 200, 500, 1000, 2000, 10000, train.size)
        var acc = 0.0
        var model = Model()

        for (i in 1 until index.size) {
            model = model.batchAdd(train.subList(index[i - 1], index[i]))
            acc = test
                    .parallelStream()
                    .map {
                        if (it.outcome == model.predict(it.inputs).maxBy { it.value }?.key) {
                            1.0
                        } else {
                            0.0
                        }
                    }
                    .toList()
                    .average()

            println(acc)
        }
        Assert.assertTrue("expected $acc > 0.83", acc > 0.83)
    }
```

File: src/test/kotlin/com/tradeshift/blayze/UCIAdultDataSetTest.kt [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        Assert.assertTrue("expected $acc > 0.83", acc > 0.83)
+    }
+
+    fun uciAdult(fname: String): List<Update> {
+        val lines = this::class.java.getResource(fname).readText(Charsets.UTF_8).split("\n")
+        val updates = mutableListOf<Update>()
+
+        for (line in lines) {
+            val split = line.split(",".toRegex()).toTypedArray()
+            if (split.size < 14) continue
```
Comment (rasmusbergpalm) [inline src/test/kotlin/com/tradeshift/blayze/UCIAdultDataSetTest.kt]: why? It should handle missing features

File: src/test/kotlin/com/tradeshift/blayze/UCIAdultDataSetTest.kt [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                    if (it.outcome == model.predict(it.inputs).maxBy { it.value }?.key) {
+                        1.0
+                    } else {
+                        0.0
+                    }
+                }
+                .toList()
+                .average()
+
+        println(acc)
```
Comment (rasmusbergpalm) [inline src/test/kotlin/com/tradeshift/blayze/UCIAdultDataSetTest.kt]: remove

File: src/test/kotlin/com/tradeshift/blayze/UCIAdultDataSetTest.kt (hunk 1 above)
Comment (liufuyang) [reply src/test/kotlin/com/tradeshift/blayze/UCIAdultDataSetTest.kt]: I remember I did this for escape some header lines in the test file:
image

**Pull request closed without merging**